src/contract/contract_function_parameters.rs

```diff
@@ -23,6 +24,25 @@ struct Argument {
     is_dynamic: bool,
 }
 
+mod private {
+    pub trait Sealed {}
+    pub trait Sized {}
```

Comment:
This trait isn't actually needed lol, `Sized` here is an empty trait, `Sized` in the other module is talking about `core::marker::Sized` (if that's even its real name), those annotations are needed to opt out of 
"self must be `Sized`" errors, but this trait isn't

Reply:
https://doc.rust-lang.org/std/marker/trait.Sized.html